parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -234,33 +234,36 @@ extractKubeBinaries() {
     local kube_binary_url="$2"
     local is_private_url="$3"
 
-    mkdir -p ${K8S_DOWNLOADS_DIR}
     local k8s_tgz_tmp_fn=${kube_binary_url##*/}
```

Comment:
nit: rename to `k8s_tgz_tmp_name`?

Reply:
fn is for filename, leave it?

Reply:
ahh, I was a bit confused since `fn` is also used to abbreviate `function`, personally I'd update it

Reply:
updated